ext/Logger.php

```diff
@@ -63,26 +65,52 @@ public function _initialize()
         if (!class_exists('\Monolog\Logger')) {
             throw new ConfigurationException("Logger extension requires Monolog library to be installed");
         }
-        $this->path = $this->getLogDir();
+    }
+
+    public static function getLogger()
+    {
+        if (self::$logger) {
+            $logger = self::$logger;
+        } else {
+            $logger = self::getDefaultLogger();
```

Comment:
Sure - I added a new Logger every time, so each test could be logged separately. Originally it was just the one Logger, but that seemed to get different outputs on the Logs, and I wanted to keep them the same. I'll take a look, see what I can come up with.